packages/openapi3/src/cli/actions/convert/generators/generate-service-info.ts

```diff
@@ -1,9 +1,13 @@
-import { TypeSpecServiceInfo } from "../interfaces.js";
+import { TypeSpecServiceInfo, TypeSpecTagMetadata } from "../interfaces.js";
 import { generateDocs } from "../utils/docs.js";
 import { generateNamespaceName } from "../utils/generate-namespace-name.js";
 import { toTspValues } from "../utils/tsp-values.js";
+import { generateTags } from "./generate-tags.js";
 
-export function generateServiceInformation(serviceInfo: TypeSpecServiceInfo): string {
+export function generateServiceInformation(
+  serviceInfo: TypeSpecServiceInfo,
+  tags: TypeSpecTagMetadata[],
```

Comment:
could this use a default value of an empty array? This would maintain backwards compatibility in the case of any consumers of the library are importing this function.